next/components/ui/UserProfile/UserProfilePassword.tsx

```diff
@@ -0,0 +1,32 @@
+import LockIcon from '@assets/images/forms/lock-white.svg'
+import UserProfileSection from '@bratislava/ui-bratislava/UserProfile/UserProfileSection'
+import UserProfileSectionHeader from '@bratislava/ui-bratislava/UserProfile/UserProfileSectionHeader'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+
+import Button from '../../forms/simple-components/Button'
+
+const UserProfilePassword = () => {
+  const { t } = useTranslation('account')
+  const { push } = useRouter()
+  return (
+    <UserProfileSection>
+      <UserProfileSectionHeader
+        title={t('password_change.title')}
+        text={t('password_change.text')}
+        isMobileColumn
+      >
+        <Button
+          variant="black"
+          startIcon={<LockIcon />}
+          size="sm"
+          text={t('password_change.button')}
+          // href="/forgotten-password"
+          onPress={() => (window.location.href = 'password-change')}
```

Comment:
because when i use href prop in our custom Button, it will lost most of Button css. its either broken (bug) or am I supposed to customise css everytime? I am trying to finish also stepper before I left, so I did it like this for now.